addons/survey/views/survey_templates_statistics.xml

```diff
@@ -417,5 +481,10 @@
             </t>
         </ul>
     </template>
+
+    <template id="survey_format_if_integer" name="Survey: remove .0">
```

Comment:
I would rename to "survey_remove_unnecessary_decimals" or something like that.
"if_integer" is very technical.